x-pack/plugins/security_solution/cypress/integration/data_sources/sourcerer.spec.ts

```diff
@@ -51,88 +56,175 @@ describe('Sourcerer', () => {
       cy.get(TOASTER).should('have.text', 'Write role required to generate data');
     });
   });
-  // Originially written in December 2020, flakey from day1
-  // has always been skipped with intentions to fix, see note at top of file
-  describe.skip('Default scope', () => {
+
+  describe('Default scope', () => {
     beforeEach(() => {
       cy.clearLocalStorage();
       loginAndWaitForPage(HOSTS_URL);
     });
 
-    it('has SIEM index patterns selected on initial load', () => {
+    it('correctly loads SIEM data view', () => {
       openSourcerer();
+      isDataViewSelection(siemDataViewTitle);
+      openAdvancedSettings();
       isSourcererSelection(`auditbeat-*`);
+      isSourcererOptions(DEFAULT_INDEX_PATTERN.filter((pattern) => pattern !== 'auditbeat-*'));
     });
 
-    it('has Kibana index patterns in the options', () => {
-      openSourcerer();
-      isSourcererOptions([`metrics-*`, `logs-*`]);
-    });
+    describe('Modified badge', () => {
+      it('Selecting new data view does not add a modified badge', () => {
+        cy.get(SOURCERER.badgeModified).should(`not.exist`);
+        openSourcerer();
+        cy.get(SOURCERER.badgeModifiedOption).should(`not.exist`);
+        openDataViewSelection();
+        isKibanaDataViewOption(dataViews);
+        cy.get(SOURCERER.selectListDefaultOption).should(`contain`, siemDataViewTitle);
+        cy.get(SOURCERER.selectListOption).contains(dataViews[1]).click();
+        isDataViewSelection(dataViews[1]);
+        saveSourcerer();
+        cy.get(SOURCERER.badgeModified).should(`not.exist`);
+        openSourcerer();
+        cy.get(SOURCERER.badgeModifiedOption).should(`not.exist`);
+      });
 
-    it('selected DATA_VIEW gets added to sourcerer', () => {
-      setSourcererOption(`metrics-*`);
-      openSourcerer();
-      isSourcererSelection(`metrics-*`);
+      it('shows modified badge when index patterns change and removes when reset', () => {
```

Comment:
I would split this into 2 different tests, something like:
`shows modified badge when index patterns change` and `removes badge when reset index pattherns`